auto_nag/scripts/workflow/no_severity.py

```diff
@@ -96,12 +95,39 @@ def set_people_to_nag(self, bug, buginfo):
     def get_bz_params(self, date):
         fields = ["triage_owner", "flags"]
         params = {
-            "bug_type": "defect",
             "include_fields": fields,
+            "email1": "intermittent-bug-filer@mozilla.bugs",
+            "emailreporter1": 1,
+            "emailtype1": "notequals",
+            "email2": "wptsync@mozilla.bugs",
+            "emailreporter2": 1,
+            "emailtype2": "notequals",
             "resolution": "---",
-            "f1": "priority",
+            "f1": "component",
             "o1": "equals",
-            "v1": "--",
+            "v1": "untriaged",
+            "f2": "OP",
+            "f3": "cf_status_firefox_nightly",
+            "o3": "equals",
+            "v3": "---",
+            "f4": "cf_status_firefox_beta",
+            "o4": "equals",
+            "v4": "---",
+            "f5": "cf_status_firefox_release",
+            "o5": "equals",
+            "v5": "---",
+            "f6": "cf_status_firefox_esr",
+            "o6": "equals",
+            "v6": "---",
+            "f7": "bug_type",
+            "o7": "equals",
+            "v7": "defect",
+            "f8": "CP",
+            "f9": "bug_severity",
+            "o9": "equals",
+            "v9": "--",
+            "j_top": "OR",
+            "limit": "0"
         }
```

Comment:
at line 117 (before your patch) params are updated so your must update this part too: some search params are added according to moment we're:
 - "first" stands for we're between 2 and 4 weeks after the product/component has been set
 - "second" stands for we're beyond 4 weeks...
So the params number "f#", "o#", ... must be changed.